src/scripts/main.js

```diff
@@ -1,3 +1,120 @@
 'use strict';
 
-// write code here
+'use strict';
```

Comment:
The 'use strict' directive is repeated. It is only necessary to declare it once at the top of the file.